charts/shoot-core/components/charts/network-policies/templates/allow-from-to-nginx.yaml

```diff
@@ -0,0 +1,21 @@
+kind: NetworkPolicy
```

Comment:
I don't see this network policy is used by the nginx-ingress-controller yet?

Reply:
i use following pod selector in this network policy file 
```
spec:
  podSelector:
    matchLabels:
      app: nginx-ingress
```

Reply:
I see, thanks 👍